### PR TITLE
feat(core): tabs defaultTab and selectDefaultOnTabsChange

### PR DESCRIPTION
**Pull request by platon-rov**
## Related Issue(s)

Refers to DXP.

## Description

* `defaultTab` input property to be able to specify the default tab.
* `selectDefaultOnTabsChange` input property to be able to control tab selection behavior on tabs change.

### After:

image

### CODE REVIEW

File: libs/core/src/lib/tabs/tab-panel/tab-panel.component.ts [hunk 1]
```diff
@@ -134,6 +141,17 @@ export class TabPanelComponent implements OnChanges {
...
+            setTimeout(() => {
+                this._updateHost();
+            });
         }
     }
+
+    /** @hidden */
+    private _updateHost(): void {
+        this._expandedClass = this.expanded;
+        this._expandedAria = this.expanded ? true : null;
```
Comment (InnaAtanasova) [inline libs/core/src/lib/tabs/tab-panel/tab-panel.component.ts:155]: why `null` but not `false`?

Comment (platon-rov) [reply libs/core/src/lib/tabs/tab-panel/tab-panel.component.ts:155]: when binding to attribute null evaluates to "no attribute at all", when false only about setting value.

File: libs/core/src/lib/tabs/tab-panel/tab-panel.component.ts [hunk 2]
```diff
@@ -134,6 +141,17 @@ export class TabPanelComponent implements OnChanges {
             }
 
             this._changeDetRef.detectChanges();
+
+            // Done this way to avoid "NG0100: Expression has changed after it was checked"
```
Comment (g-cheishvili) [inline libs/core/src/lib/tabs/tab-panel/tab-panel.component.ts:145]: ngZone.onStable?

**Review by mikerodonnell89 — COMMENTED** [review]
The `defaultTab` input looks good and makes sense, but I am confused by what exactly the next example (screenshot below) is supposed to do, can you expand on the description a bit?
Screen Shot 2022-08-23 at 10 10 29 AM

**Review by InnaAtanasova — APPROVED** [review]

**Review by mikerodonnell89 — APPROVED** [review]

**Pull request merged**